### ISSUE TITLE
Need to update comments for Linux re. libcurl

### ISSUE THREAD

**Issue by linuxgurugamer** [KSP-CKAN/CKAN-support#109]
For people who use Fedora and other RPM-based systems, your instructions are not correct.

Also, once it displays it once, it doesn't display again and there doesn't seem to be any way to redisplay the notes.

For Fedora 21 (and other RPM systems):  yum install libcurl

**Comment by linuxgurugamer** [comment]
the correct command for Fedora 21 is:  yum install libcurl-devel

**Comment by pjf** [comment]
Thank you! I've updated [the release notes](https://github.com/KSP-CKAN/CKAN/releases/tag/v1.6.6) which is what the auto-updater uses.

We *should* be able to detect when the supporting libcurl libraries aren't installed and notify the user, but that'll be as part of the great CurlSharpening of the core. :)  A 'display release notes' option for the current release is a good idea, I've opened KSP-CKAN/CKAN-GUI#71 to track that.

Many thanks!

~ Paul
